### DISCUSSION TITLE
Laravel Reverb for the subscription

### DISCUSSION THREAD

**Discussion by wmt-web-pruthvip** [nuwave/lighthouse#2533] [General]
Hey Guys,

I am trying to use laravel reverb for the graphql subscription. but in that I am facing issue. As it is based on echo so I have used a provider as echo. Does anyone has implemented it?

I am facing an issue as [here](https://github.com/laravel/reverb/issues/120)

**Comment by michaelLoeffelmann** [comment]
Hi,

have you solved your problem yet? I'm running in the same issue and have no idea to solve it.

**Reply by wmt-web-pruthvip** [reply]
Hey there!

Echo and reverb use different setups, so using echo as a temporary fix for reverb wouldn't work. On top of that, reverb drivers aren't even available yet.

So, for now, it looks like echo is the way to go until they release the reverb driver.

**Reply by michaelLoeffelmann** [reply]
I hope that there are Plans to create a driver for reverb :-)

when using echo, what's the package for the server-part to provide the websocket server?

**Comment by stayallive** [comment]
Use the Pusher driver instead, Laravel Reverb is Pusher API compatible.
